extra/peg/javascript/tokenizer/tokenizer-tests.factor

```diff
@@ -23,3 +23,10 @@ IN: peg.javascript.tokenizer.tests
 { V{ T{ ast-regexp f "<(w+)[^>]*?)/>" "g" } } } [
   "/<(\\w+)[^>]*?)\\/>/g" tokenize-javascript
 ] unit-test
+
+{
+  V{
+    T{ ast-name { value "abc" } }
+    T{ ast-string { value "def" } }
+  }
+} [ "abc\"def\"" tokenize-javascript ]
```

Comment:
Well, of course it doesn't. *facepalm*

Next time I'll try *running* the tests first...